src/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -27,17 +27,16 @@
 import com.owncloud.android.lib.common.utils.Log_OC;
 
 import java.io.File;
+import java.io.Serializable;
```

Comment:
Removed.
I have now enabled "optimize imports" on git commit in android studio.